pontoon/base/models.py

```diff
@@ -94,6 +93,16 @@ def cldr_plurals_list(self):
         else:
             return map(int, self.cldr_plurals.split(','))
 
+    @classmethod
+    def cldr_plural_to_id(self, cldr_plural):
+        for i in self.CLDR_PLURALS:
```

Comment:
The only other Locale.CLDR_PLURALS used is in `validate_cldr`, and the way it's used there and here could be simplified if you made it a dict instead of a list. Then you could just use `.get()` to fetch the string and `in` to check if the integer is valid.